.github/workflows/ci.yml

```diff
@@ -0,0 +1,65 @@
+name: CI
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+    runs-on: windows-latest
```

Comment:
I think we should do this for multiple images since the repo support back to rs5

Reply:
I think the only available windows image type is ws2019 (which is RS5): https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources

Reply:
@dcantah is right. If you want this to be changed to `windows-2019`, just say so and I'll update PR :)

Reply:
ah okay, I see. I think changing it to `windows-2019` would be better then just to make it more explicit.

Reply:
Done